### ISSUE TITLE
SingleLevelBomAsSpecified not usable

### ISSUE THREAD

**Issue by ds-jhartmann** [eclipse-tractusx/item-relationship-service#687]
**As** ... ,
**I want** ... ,
**so that** ... .

## Hints / Details

- Semantic Model SingleLevelBomAsSpecified does not contain a BPN for the childItems
- see https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.single_level_bom_as_specified/2.0.0

## Blocked By
- New version of SingleLevelBomAsSpecified with BPN relationship is released

## Acceptance Criteria

- [ ] New version of SingleLevelBomAsSpecified with BPN relationship is implemented

## Out of Scope

- ...
